setup.py

```diff
@@ -160,6 +157,9 @@
         "jsonschema == 2.4.0",
         "klein == 0.2.3",
         "pyrsistent == 0.9.1",
+
+        # This is used for provisioning code.
+        "effect==0.1a13",
```

Comment:
This probably belongs better in the branch for https://clusterhq.atlassian.net/browse/FLOC-1099 but adding this here is easier than adjusting the tests to skip here (since it is going to get added anyway).